fizzbuzz.js

```diff
@@ -0,0 +1,80 @@
+//Really, really inefficient way to do FizzBuxx
+
+//Variable declaration
+var maxNum = 100;
+var primeArray = [];
+var fizzBuzzArray = [];
+var tempArray = [];
+var i = 2;
+var x;
+var prime;
+var prime2;
+var count = 0;
+var print;
+
+//Calculates all prime numbers between 2 and maxNum (inclusive on both ends)
+while (i <= maxNum) {
+    prime = true;
+    for (x in primeArray) {
+        if (i % primeArray[x] == 0) {
+            prime = false;
+        }
+    }
+    if (prime == true) {
+        primeArray.push(i);
+    }
+    i++;
+}
+
+//Removes 3 and 5 from the prime numbers
+primeArray.splice(1,2);
+
+//Adds the prime numbers to the array that is printed at the end
+for (print in primeArray) {
+    (fizzBuzzArray[primeArray[print] - 1] = primeArray[print]);
+}
+
+/*For each prime in primeArray, takes it times all other primes in the array,
+then times all values received from that by all primes in primeArray until the
+smallest number received from this multiplication is greater than maxNum, adding
+each number received that is smaller than maxNum to the array to be printed*/
+for (prime in primeArray) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in

> Note: for...in should not be used to iterate over an Array where the index order is important.
Array indexes are just enumerable properties with integer names and are otherwise identical to general Object properties. There is no guarantee that for...in will return the indexes in any particular order.  The for...in loop statement will return all enumerable properties, including those with non–integer names and those that are inherited.

> Because the order of iteration is implementation-dependent, iterating over an array may not visit elements in a consistent order. Therefore it is better to use a for loop with a numeric index (or Array.prototype.forEach() or the for...of loop) when iterating over arrays where the order of access is important.